### PR TITLE
Image issue fixes

### PR DESCRIPTION
**Pull request by dinesh-mobi**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by dinesh-mobi** [comment]
Image issue/error that was in failure report has been fixed, if any issue is still there please write in details.

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-06-09 at 17 05 58

File: app/assets/images/payment_icons/fortiva.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-fortiva" width="38" height="24" role="img"><title id="pi-fortiva">Fortiva</title><g fill="none" fill-rule="evenodd"><rect fill-opacity=".07" fill="#000" width="38" height="24" rx="3"/><rect fill="#FFF" x="1" y="1" width="36" height="22" rx="2"/><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3Z" fill="#000" fill-rule="nonzero" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32Z" fill="#FFF" fill-rule="nonzero"/><path d="M25 6.818C23.532 5.093 21.393 4 19.025 4 14.593 4 11 7.795 11 12.476c0 2.62 1.132 4.97 2.9 6.524l2.292-10.104c.301-1.28 1.052-2.079 2.327-2.078H25" fill="#E5842D"/><path d="M15 19.086a8.229 8.229 0 0 0 3.77.914c4.546 0 8.23-3.688 8.23-8.236 0-1.364-.32-2.631-.907-3.764-.345.21-.773.349-1.269.352l-5.505.03c-.423.003-1.232.009-1.456.982l-.523 2.529 6.276-.041c-.263 1.01-1.255 2.298-2.526 2.269l-3.159.02c-.423.004-1.232.046-1.379.979-.392 1.947-.75 3.266-1.535 3.956" fill="#889591"/></g></svg>
```
[the remark this answers is not in the document]
Comment (dinesh-mobi) [reply app/assets/images/payment_icons/fortiva.svg]: I have fixed the issue as per suggestion and pushed

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
👋 Hello, your icon(s) have been released in the latest version of this library. The rollout process will begin within a few days. We wanted to have it/them live this week but looking more likely to be live early to mid next week. Will report back here when live. Thank you for your patience. 🙇‍♂️

**Comment by larouxn** [comment]
👋 Hello, actually your icon(s) should be live everywhere now. 🚀

**Pull request merged**